omnibus/package-scripts/agent/prerm

```diff
@@ -123,11 +123,37 @@ remove_custom_integrations()
     fi
 }
 
+remove_version_history()
+{
+    # Since 6.22.0/7.22.0, a file containing the version history of the currently installed
+    # Agent is created (by default in /opt/datadog-agent/run). On a full uninstall, remove this
+    # file.
+    # This is a best-effort solution, as users can decide to put this file in another place
+    # by changing the logs_config.run_path value.
+
+    if [ -f "$INSTALL_DIR/run/version-history.json" ]; then
+        echo "Removing version history file"
+        rm "$INSTALL_DIR/run/version-history.json"
+    fi
```

Comment:
I am not sure on what happens if the file exists but you don't have permission to delete it. I think it is safer to do
```suggestion
    if [ -f "$INSTALL_DIR/run/version-history.json" ]; then
        echo "Removing version history file"
        rm "$INSTALL_DIR/run/version-history.json" || true
    fi
```
just in case

Reply:
On one hand, I agree, but on the other hand, this means root cannot remove the file, which is a problem by itself (given that this file is not supposed to be in use when this script runs). 

Note that all other file removals we do in the script are also not guarded (eg. when removing integrations), so for consistency we'd need to add a guard to them too.

Reply:
It is a weird edge case but if we are assuming customers will move the file or modify it I think it's not a bad idea to add the guard. We could add it on the other removals for consistency on a different PR.